modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorBalloonEditorConfigContributor.java

```diff
@@ -56,87 +42,6 @@ public void populateConfigJSONObject(
 		super.populateConfigJSONObject(
```

Comment:
We still needing CKEditorBalloonEditorConfigContributor class to provide the dynamic values like: links to DL, portal language, direction, resolved CSS info, [etc.](https://liferay.atlassian.net/wiki/spaces/~73202423/pages/1810106835/Provide+editor+configuration+in+JSON+files#Required-Server-Side-Properties)